### ISSUE TITLE
Showing verbose policy version incorrectly displays rule matches when using NOT

### ISSUE THREAD

**Issue by xelwarto** [akamai/cli-visitor-prioritization#4]
Given a VP policy version that has a rule that matches on the following:

- URL Path is one of **/testpath/***
AND
- URL Path is not one of **/testpath/new/***

When viewing (verbose) the policy version using the CLI, the rule incorrectly displays the NOT path match and makes it look like the rule is matching all paths.

````
Rule Name: Cloud Throttle
  Traffic Percentage: 100.0

  Match Type: path
  Match Value: /testpath/*
  AND
  Match Type: path
  Match Value: /testpath/new/*

  Rule is DISABLED
````

**Comment by xelwarto** [comment]
Command used to produce error:

````
akamai show --policy testpolicy --version 1 --verbose
````

**Comment by vreddhi** [comment]
This is being fixed. Should be available in next release.

**Comment by vreddhi** [comment]
This is now fixed and available in new release 0.2.0
